src/main/java/uk/gov/ons/ctp/integration/contactcentresvc/service/impl/CaseServiceImpl.java

```diff
@@ -432,71 +419,13 @@ private Case getCaseFromDb(long caseRef) throws CTPException {
   private RefusalDetails createRespondentRefusalPayload(
       UUID caseId, RefusalRequestDTO refusalRequest) throws CTPException {
 
-    // Create message payload
     RefusalDetails refusal = new RefusalDetails();
-    refusal.setType(mapToType(refusalRequest.getReason()));
-    CollectionCaseCompact collectionCase = new CollectionCaseCompact(caseId);
-    refusal.setCollectionCase(collectionCase);
-    refusal.setAgentId(Integer.toString(refusalRequest.getAgentId()));
-    refusal.setCallId(refusalRequest.getCallId());
-    refusal.setHouseholder(refusalRequest.getIsHouseholder());
-
-    // Populate contact
-    ContactCompact contact = createRefusalContact(refusalRequest);
-    refusal.setContact(contact);
-
-    // Populate address
-    AddressCompact address = new AddressCompact();
-    address.setAddressLine1(refusalRequest.getAddressLine1());
-    address.setAddressLine2(refusalRequest.getAddressLine2());
-    address.setAddressLine3(refusalRequest.getAddressLine3());
-    address.setTownName(refusalRequest.getTownName());
-    address.setPostcode(refusalRequest.getPostcode());
-    Region region = refusalRequest.getRegion();
-    if (region != null) {
-      address.setRegion(region.name());
-    }
-    UniquePropertyReferenceNumber uprn = refusalRequest.getUprn();
-    if (uprn != null) {
-      address.setUprn(Long.toString(uprn.getValue()));
-    }
-    refusal.setAddress(address);
+    refusal.setCaseId(caseId);
+    refusal.setType(refusalRequest.getReason().name());
 
     return refusal;
   }
 
-  private ContactCompact createRefusalContact(RefusalRequestDTO refusalRequest) {
-    ContactCompact contact = null;
-
-    if (refusalRequest.getReason() == Reason.HARD) {
-      contact = new ContactCompact();
-      contact.setTitle(encrypt(refusalRequest.getTitle()));
-      contact.setForename(encrypt(refusalRequest.getForename()));
-      contact.setSurname(encrypt(refusalRequest.getSurname()));
-    }
-    return contact;
-  }
-
-  private String encrypt(String clearValue) {
-    if (clearValue == null) {
-      return null;
-    }
-    List<Resource> keys = List.of(appConfig.getPublicPgpKey1(), appConfig.getPublicPgpKey2());
-    String encStr = PgpEncrypt.encrypt(clearValue, keys);
```

Comment:
Since we have removed PgpEncrypt usage, we can remove dependency, code, and unit tests for this

Reply:
Deleted